data/json/recipes/recipe_food.json

```
@@ -1055,6 +1127,7 @@
     "autolearn": true,
     "qualities": [ { "id": "COOK", "level": 2 } ],
```

```suggestion
    "qualities": [ { "id": "CUT", "level": 1 }, { "id": "COOK", "level": 2 } ],
```
Dogfood would need cut ingredients.